packages/round-manager/src/features/api/utils.ts

```diff
@@ -224,4 +224,4 @@ export const generateApplicationSchema = (metadata: any): Array<SchemaQuestion>
 }
 
 // Checks if tests are being run jest
-export const isJestRunning = () =>  process.env.JEST_WORKER_ID !== undefined;
```

Comment:
we found the lit-sdk issue! fixed in https://github.com/gitcoinco/grants-round/pull/317/commits/c45759aed555d007fb26bb949b895f54fde37f1d